jstz_node/src/services/accounts.rs

```diff
@@ -29,11 +31,63 @@ async fn nonce(
     Ok(HttpResponse::Ok().json(nonce))
 }
 
+#[get("/{address}/kv/{key:.*}")]
```

Comment:
Naive question: What is the `.*` for?

Reply:
Key can be empty - .* ensures that it is matched even then.

Reply:
Keys cannot be empty, I've just checked. 

For example, if you deploy:
```js
const KEY = "";

const handler = () => {
  let counter = Kv.get(KEY);
  console.log(`Counter: ${counter}`);
  if (counter === null) {
    counter = 0;
  } else {
    counter++;
  }
  Kv.set(KEY, counter);
  return new Response();
};

export default handler;
```
and then execute it, you obtain a `Err("EvalError: PathError: PathError::InvalidEmptyStep")` in the receipt.